src/facade/cmd_arg_parser.h

```diff
@@ -14,12 +14,7 @@ namespace facade {
 
 // Utility class for easily parsing command options from argument lists.
 struct CmdArgParser {
-  enum ErrorType {
-    OUT_OF_BOUNDS,
-    SHORT_OPT_TAIL,
-    INVALID_INT,
-    INVALID_CASES,
-  };
+  enum ErrorType { OUT_OF_BOUNDS, SHORT_OPT_TAIL, INVALID_INT, INVALID_CASES, INVALID_NEXT };
```

Comment:
Btw, an area where the parser actually needs improvement is error reporting

Reply:
will think about it